server/src/test/scala/com/ossum/riddl/lsp/server/RiddlLSPTextDocumentSpec.scala

```diff
@@ -120,12 +113,12 @@ class RiddlLSPTextDocumentSpec
       with CompletionRequestSpec {
       position.setLine(errorLine)
       position.setCharacter(errorCharOnLine)
-      makeRequest()
+      requestCompletion()
 
-      whenReady(completionResultF.asScala) { eitherList =>
-        eitherList.isRight mustBe true
-        eitherList.getRight.getItems.asScala.length mustEqual 1
-        eitherList.getRight.getItems.asScala.head.getDetail mustEqual
+      whenReady(completionResultF.asScala) { completionList =>
+        completionList.isRight mustBe true
```

Comment:
This would be easier to read if you used a case clause (implied match on the completionList)

Reply:
The compiler seems to ask for type information, which is java-based and here for completions is a long Either. This seems less verbose, and I'm copy pasting these lines anyway

Reply:
how can an expression as the result of "asScala" be "java-based" ?

Reply:
`asScala` operates only on the top-level type of the object, here a Future. The Future contains an `lsp4j.jsonrpc.messages.Either[java.util.List[CompletionItem],CompletionList]` where, ironically and sadly, `CompletionList` contains a field `items: java.util.List[CompletionItem]`. Most likely this `Either` is here for backwards compatibility in the API

Reply:
Ugh. Java APIs. Fell free to resolve.